test/UnitLoad.cpp

```diff
@@ -180,7 +181,7 @@ UnitBase::TestResult UnitLoad::testExcelLoad()
 
         // Expected format is something like 'status: type=text parts=2 current=0 width=12808 height=1142 viewid=0\n...'.
         StringVector tokens(StringVector::tokenize(status, ' '));
-        LOK_ASSERT_EQUAL(static_cast<size_t>(7), tokens.size());
+        LOK_ASSERT_EQUAL(static_cast<size_t>(9), tokens.size());
```

Comment:
above comment with example content has 7 tokens, would be good to update

Reply:
how it become 9 now? this check wasn't executed in the past?

Reply:
I updated it somewhere else and missed this one. Will update in a follow up patch.

Yes, this test was disabled. I enabled it and 4 or 5 others.

Thanks!